acceptance/testdata/analyzer/analyze-image/container/cnb/local-bionic-stack.toml

```diff
@@ -0,0 +1,6 @@
+[run-image]
+ image = "localcompany/stack:bionic"
+ mirrors = []
+[build-image]
+ stack-id = "io.buildpacks.stacks.bionic"
+ mixins = []
```

Comment:
Would it make sense to move this and /cnb/missing-run-image.toml to the Dockerfile, so that all the stack tomls are defined in one place? (Having it defined in two places caused me a bit of confusion)

Reply:
11a5a76